src/routes/files/handlers.js

```diff
@@ -111,11 +121,20 @@ export default async function uploadHandler(req, res) {
     }
     try {
       await s3Uploader(buffer, fileName, type);
-      await updateStatus(metadata.id, 'UPLOADED');
+      await updateStatus(metadata.id, fileStatuses.uploaded);
       res.status(200).send({ id: metadata.id });
     } catch (err) {
       if (metadata) {
-        await updateStatus(metadata.id, 'UPLOAD_FAILED');
+        await updateStatus(metadata.id, fileStatuses.uploaded);
+      }
+      await handleErrors(req, res, err, logContext);
+    }
+    try {
+      await addToScanQueue({ key: metadata.key });
+      await updateStatus(metadata.id, fileStatuses.scanning);
+    } catch (err) {
+      if (metadata) {
+        await updateStatus(metadata.id, fileStatuses.uploaded);
```

Comment:
Should this also be `uploadFailed`?

Reply:
No, the upload succeeded, We don't currently have a status for tried to add to the queue, but I was planning on adding that to a later pr, but I guess I could add it now along with tests.

Reply:
I am gonna add two new statuses `SCANNING_QUEUED` for when it has been added to the queue, and `QUEUEING_FAILED` for when the queue is unavailable.